### ISSUE TITLE
sed: can't read /u01/oracle/create-wls-domain.py: Permission denied

### ISSUE THREAD

**Issue by wggit** [oracle/docker-images#649]
My images
```
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
12213-domain        latest              2942b236ab9e        5 minutes ago       2.81 GB
oracle/weblogic     12.2.1.3-generic    b0945a44b0c2        17 hours ago        2.81 GB
oracle/serverjre    8                   60a107ea6353        7 days ago          269 MB
oraclelinux         7-slim              a6e9e9e5ddc1        5 weeks ago         118 MB

docker run -p 7001:7001 --name=wlsadmin --hostname wlsadmin -e ADMIN_USERNAME=weblogic -e ADMIN_PASSWORD=<passwd> 12213-domain:latest

      ----> 'weblogic' admin password: <passwd>
sed: can't read /u01/oracle/create-wls-domain.py: Permission denied
Initializing WebLogic Scripting Tool (WLST) ...
Welcome to WebLogic Server Administration Scripting Shell
Type help() for help on available commands
Problem invoking WLST - java.io.FileNotFoundException: /u01/oracle/create-wls-domain.py
/u01/oracle/createAndStartWLSDomain.sh: line 48: /u01/oracle/user_projects/domains/base_domain/bin/setDomainEnv.sh: No such file or directory
/u01/oracle/createAndStartWLSDomain.sh: line 53: /u01/oracle/user_projects/domains/base_domain/startWebLogic.sh: No such file or directory
touch: cannot touch '/u01/oracle/user_projects/domains/base_domain/servers/AdminServer/logs/AdminServer.log': No such file or directory
tail: cannot open '/u01/oracle/user_projects/domains/base_domain/servers/AdminServer/logs/AdminServer.log' for reading: No such file or directory
tail: no files remaining
```

If I manually copy create-wls-domain.py to container folder, AdminServer gets created and runs fine.

**Comment by wggit** [comment]
This issue can not be reproduced if you use oracle/weblogic     12.2.1.3-generic to create domain and AdminServer.

**Comment by mriccell** [comment]
@wggit  You are not mapping the domain home outside the image with -v.
docker run -d --name wlsadmin --hostname wlsadmin -p 7001:7001 --env-file ./container-scripts/domain.properties -e ADMIN_PASSWORD=<admin_password> -v <host directory>:/u01/oracle/user_projects 12213-domain

**Comment by wggit** [comment]
Thank you @mriccell . I tried that option too but getting same error.
u01 is local file system. Folder /u01/oracle/user_projects exists.

**Comment by wggit** [comment]
I don't see $PRE_DOMAIN_HOME is getting created as per Dockerfile which may be root cause.

**Comment by mriccell** [comment]
fixed in PR #662

**Comment by kerberos5** [comment]
Hi,
my images:
docker@ubuntu:~/kubernates/OracleWebLogic/samples/12213-domain$ docker images
REPOSITORY                           TAG                  IMAGE ID            CREATED             SIZE
12213-weblogic-domain-in-volume      latest               90288122f467        7 seconds ago       961MB
oracle/weblogic                      12.2.1.3-developer   5967bd051ce6        23 hours ago        961MB
<none>                               <none>               3495a2b2c780        23 hours ago        1.45GB

when start my container whit . ./run_managed_server.sh run but i have into logs container:
docker@ubuntu:~/kubernates/OracleWebLogic/samples/12213-domain$ docker logs AdminContainer 
Domain Home is:  /u01/oracle/user_projects/domains/myDomain
/u01/oracle/properties/domain_security.properties
/u01/oracle/properties/domain.properties
Java Options: -Doracle.jdbc.fanEnabled=false -Dweblogic.StdoutDebugEnabled=false
/u01/oracle/container-scripts/createWLSDomain.sh: line 85: /u01/oracle/user_projects/domains/myDomain/bin/setDomainEnv.sh: Permission denied
Starting the Admin Server
==========================
/u01/oracle/container-scripts/createWLSDomain.sh: line 91: /u01/oracle/user_projects/domains/myDomain/startWebLogic.sh: Permission denied

help please!
thanks
D.
